navigation/mapping_exploration/CMakeLists.txt

```
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(mapping_exploration)
```

Just to be consistent with the rest of the packages:

```suggestion
project(ca_mapping_exploration)
```

---

Done!